olm-catalog/serverless-operator/generate_csv.sh

```diff
@@ -29,27 +29,21 @@ images=(
   ["eventing-webhook__eventing-webhook"]="${eventing}-webhook"
   ["storage-version-migration-eventing__migrate"]="${eventing}-storage-version-migration"
 
-  # the next three images have 0 replicas, they got removed, and point to 0.15 release images to migrate their deployments to 0
-  ["broker-controller__broker-controller"]="${registry}/knative-v0.15.2:knative-eventing-channel-broker"
-  ["broker-filter__filter"]="${registry}/knative-v0.15.2:knative-eventing-broker-filter"
-  ["broker-ingress__ingress"]="${registry}/knative-v0.15.2:knative-eventing-broker-ingress"
-
-  # the mt broker replaces the removed broker
   ["mt-broker-controller__mt-broker-controller"]="${eventing}-mtchannel-broker"
   ["mt-broker-filter__filter"]="${eventing}-mtbroker-filter"
   ["mt-broker-ingress__ingress"]="${eventing}-mtbroker-ingress"
   ["imc-controller__controller"]="${eventing}-channel-controller"
   ["imc-dispatcher__dispatcher"]="${eventing}-channel-dispatcher"
 
-  ["v0.16.0-broker-cleanup__brokers"]="${eventing}-broker-cleanup"
+  ["v0.17.0-pingsource-cleanup__pingsource"]="${eventing}-pingsource-cleanup"
```

Comment:
Shouldn't this be 0.17.2?

Reply:
Nope, unfortunately the name of the job is hard coded to `0.17.0` on the upstream `release-0.17` branch:
https://github.com/knative/eventing/blob/master/config/post-install/v0.17.0/pingsource-cleanup.yaml#L4

However, the image does contain the 0.17.2 build that we do:
https://github.com/openshift-knative/serverless-operator/pull/535/files#diff-f70d314a9f9d720fb31144a40e937449R299

@markusthoemmes I can change it in our operator fork to `v0.17` instead, if you feel like changing. :man_shrugging:

Reply:
Got it, thanks. WFM if it's a mistake upstream anyway, we can keep it as-is.